packages/palette/src/elements/Select/Select.tsx

```diff
@@ -73,11 +73,11 @@ export const SmallSelect: SFC<SelectProps> = props => {
 }
 
 /**
- * A mini drop-down select menu
+ * A small version of drop-down select menu
  */
-export const MiniSelect: SFC<SelectProps> = props => {
+export const SelectSmall: SFC<SelectProps> = props => {
```

Comment:
@oxaudo - can you add a comment about deprecating the `SmallSelect` component just up above? It seems a bit confusing having these things mixed together. (Also, maybe `SelectSmall` can be moved into its own file.)

Reply:
Totally. Left a comment. And I do want to work on replacing this SmallSelect next and will kill it here once that is done.

I think it might be ok to keep the implementation in the same file since those are reusing the props interface and couple of other css definitions.